pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -50,7 +50,25 @@ msgctxt "#30007"
 msgid "Response timeout in seconds"
 msgstr ""
 
-#empty strings from id 30008 to 30099
+msgctxt "#30008"
+msgid "Advanced Settings"
```

Comment:
Wait, never mind, it seems we've used this scheme previously too (e.g. "Data Transfer").